cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DefaultNamespaceAdmin.java

```diff
@@ -122,11 +122,7 @@ public NamespaceMeta load(NamespaceId namespaceId) throws Exception {
     });
     String masterPrincipal = cConf.get(Constants.Security.CFG_CDAP_MASTER_KRB_PRINCIPAL);
     try {
-      if (SecurityUtil.isKerberosEnabled(cConf)) {
```

Comment:
There is one more place in `DatasetServiceClient` where similar check is made. Should we remove that too?